ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -80,17 +83,19 @@ function WorkloadCvesOverviewPage() {
                         Prioritize and manage scanned CVEs across images and deployments
                     </FlexItem>
                 </Flex>
-                <FlexItem className="pf-u-pr-lg">
-                    <Button
-                        variant="secondary"
-                        onClick={() => {
-                            setDefaultWatchedImageName('');
-                            watchedImagesModalToggle.openSelect();
-                        }}
-                    >
-                        Manage watched images
-                    </Button>
-                </FlexItem>
+                {canWriteWatchedImages && (
+                    <FlexItem className="pf-u-pr-lg">
```

Comment:
Outside of these changes, the utility class caught my eye.

The alignment of the button with the inner white section below it makes total sense, but I was curious why why `PageSection` has `'noPadding'` and then it seems like its descendants specify the padding.

Reply:
Good question - I'm not sure what the original reason was but it looks like it is no longer valid. I removed the `noPadding` prop and the utility classes and the layout was identical. (At least on larger screen sizes; the padding is less on tablet and lower sized devices, but is consistent with the PatternFly defaults.)